<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TerminalPane</class>
 <widget class="QWidget" name="TerminalPane">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>259</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolBar" name="toolBar">
     <property name="movable">
      <bool>false</bool>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <addaction name="actionRemove"/>
     <addaction name="separator"/>
     <addaction name="actionClear"/>
     <addaction name="actionZoom_In"/>
     <addaction name="actionZoom_Out"/>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::East</enum>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <property name="tabBarAutoHide">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionRemove">
   <property name="icon">
    <iconset resource="../../r.qrc">
     <normaloff>:/Resource/icons/Trash_16x.svg</normaloff>:/Resource/icons/Trash_16x.svg</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="../../r.qrc">
     <normaloff>:/Resource/icons/AddImmediateWindow_16x.svg</normaloff>:/Resource/icons/AddImmediateWindow_16x.svg</iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="../../r.qrc">
     <normaloff>:/Resource/icons/CleanData_16x.svg</normaloff>:/Resource/icons/CleanData_16x.svg</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="icon">
    <iconset resource="../../r.qrc">
     <normaloff>:/Resource/icons/ZoomIn_16x.svg</normaloff>:/Resource/icons/ZoomIn_16x.svg</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="icon">
    <iconset resource="../../r.qrc">
     <normaloff>:/Resource/icons/ZoomOut_16x.svg</normaloff>:/Resource/icons/ZoomOut_16x.svg</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../r.qrc"/>
 </resources>
 <connections/>
</ui>
